GitAutoDeploy.py

```diff
@@ -435,10 +438,10 @@ def get_config(self):
             if not os.path.isdir(repo_config['path']):
 
                 print "Directory %s not found" % repo_config['path']
-                GitWrapper.clone(url=repo_config['url'], path=repo_config['path'])
+                GitWrapper.clone(url=repo_config['url'], branch=repo_config['branch'] if 'branch' in repo_config else None, path=repo_config['path'])
 
                 if not os.path.isdir(repo_config['path']):
-                    print "Unable to clone repository %s" % repo_config['url']
+                    print "Unable to clone %s branch of repository %s" % (repo_config['branch'] if 'branch' in repo_config else "default", repo_config['url'])
```

Comment:
One small thing, `default` should be renamed to `master`. Aside from that, this looks good to :ship: .